plugin/audit_log/audit_log.c

```diff
@@ -811,7 +811,53 @@ int audit_log_plugin_init(void *arg MY_ATTRIBUTE((unused)))
 
   audit_log_filter_init();
 
-  return(0);
+  if (audit_log_exclude_accounts != NULL && audit_log_include_accounts != NULL)
```

Comment:
audit_log_filer_init located in separate module and doesn't know about plugin variables. This code is aimed to validate plugin variables.